charts/fybrik/values.yaml

```diff
@@ -341,6 +364,11 @@ opaServer:
   # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
   containerSecurityContext: {}
 
+  enableReadinessProbe: true
```

Comment:
why did you add `enableReadinessProbe` and `enableLivenessProbe` to OPA server only ?

Reply:
OPA server implements the proves with http requests which require certificate verification when it uses tls as is used in notebook-test-readflow.tls. to avoid configuring kublet with the CA certificates this flag was added.
The rest of the deployments are not using probes with http requests so they can be executed anyway.

Reply:
Can you add some explanation in the file ?

Reply:
Sure, I mistakenly put it only in charts/fybrik/notebook-test-readflow.tls.values.yaml